linker/jvm/src/main/scala/org/scalajs/linker/backend/closure/ClosureLinkerBackend.scala

```diff
@@ -96,17 +98,28 @@ final class ClosureLinkerBackend(config: LinkerBackendImpl.Config)
           closureExterns.asJava, List(module).asJava, options)
     }
 
+    if (result.errors.nonEmpty) {
```

Comment:
What does `result.errors` contain? Can't we send that to the logger?

Reply:
Yeah, right. I had it all wrong. It turns out it's quite possible to send things to the logger. It's just slightly suboptimal in the ordering, but it works. See the updated code.